qemu/tests/numa_prealloc_threads.py

```diff
@@ -44,26 +58,42 @@ def run(test, params, env):
         test.fail("No thread-id setted.")
 
     expected_cpu_affinity = thread_context_device.get_param("cpu-affinity")
+    if expected_cpu_affinity is None:
```

Comment:
When will this happen? 
Sorry I didn't see when it would be None

Reply:
@zhenyzha 
Thanks for pointing this out, this part is residual code. Updated this part as well :grinning: